.github/workflows/comment-commands.yml

```
@@ -13,12 +13,11 @@ jobs:
+      issues: write
 
     steps:
-      - run: gh issue edit "${{ env.ISSUE_NUMBER }}" --add-assignee "@${{ env.USER_LOGIN }}"
-        env:
-          ISSUE_NUMBER: ${{ github.event.issue.number }}
-          USER_LOGIN: ${{ github.event.comment.user.login }}
-          GH_TOKEN: ${{ github.token }}
-          GH_REPO: ${{ github.repository }}
+      # Using REST API and not `gh issue edit`. https://github.com/cli/cli/issues/6235#issuecomment-1243487651
+      - run: curl \
+          -H "Authorization: token ${{ github.token }}" \
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 21: mapping values are not allowed in this context

(syntax-check)